.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  build-and-test:
+    name: build-and-test on ${{matrix.os}}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: | 
+            2.1.x
+            3.1.x
```

Comment:
Are there still projects that are using these? Or do we only need the 6.0 SDK?

Reply:
They are currently targeting 2.1standard and the test proj is net3.1, should I rev them higher?

Reply:
netstandard 2.1 can be done with just the 6.0 SDK so it sounds like you can at least delete 2.1.x